staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -306,6 +308,7 @@ func (s preparedGenericAPIServer) NonBlockingRun(stopCh <-chan struct{}) error {
 		}
 	}
 
+	s.installReadyz(stopCh)
```

Comment:
this is strange to install it here. I my PR I had another stop check, didn't I?

Reply:
No, it was like this, but I will plumb the stopCh logic as we discussed offline.